ReactWindows/ReactNative/Views/Scroll/ReactScrollViewManager.cs

```diff
@@ -224,6 +228,39 @@ public void SetZoomScale(ScrollViewer view, bool? enabled)
                 : ZoomMode.Disabled;
         }
 
+        /// <summary>
+        /// Sets the tab navigation for the view.
+        /// </summary>
+        /// <param name="view">The view.</param>
+        /// <param name="tabNavigation">The tab navigation.</param>
+        [ReactProp("tabNavigation")]
+        public void SetTabNavigation(ScrollViewer view, string tabNavigation)
+        {
+            view.TabNavigation = EnumHelpers.ParseNullable<KeyboardNavigationMode>(tabNavigation) ?? KeyboardNavigationMode.Local;
+        }
+
+        /// <summary>
+        /// Disables keyboaed based arrow scrolling.
```

Comment:
>keyboaed [](start = 21, length = 8)

typo

Reply:
fixed

---
In reply to: [156582755](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156582755) [](ancestors = 156582755)